src/toDate/index.js

```diff
@@ -1,5 +1,4 @@
-import toInteger from 'date-fns/_lib/toInteger/index.js'
-import getTimezoneOffsetInMilliseconds from 'date-fns/_lib/getTimezoneOffsetInMilliseconds/index.js'
+import { getTimezoneOffsetInMilliseconds } from 'date-fns/_lib/getTimezoneOffsetInMilliseconds.js'
```

Comment:
I tried your fork on my ESM project and this import fails because it is not exported by date-fns

```
Error [ERR_PACKAGE_PATH_NOT_EXPORTED]: 
Package subpath './_lib/getTimezoneOffsetInMilliseconds.js' is not defined by "exports" in 
/../date-fns-tz/node_modules/date-fns/package.json 
imported from 
/../date-fns-tz/esm/toDate/index.js
```

I can patch date-fns to export it, but not sure what would be a better long term solution

Reply:
@Cellule did you find a resolution for this? I have the same error.